Client/States/GameplayState.cpp

```diff
@@ -725,3 +730,22 @@ float GameplayState::CalculateCompletion(Vector3 playerCurPos){
     auto progress = playerCurPos - startPos;
     return progress.Length()/levelLen;
 }
+
+void GameplayState::RenderFlairObjects(){
```

Comment:
What is a flair object? Like just a decorative one?

Reply:
Yeah, just objects that are decor. This is what would be used if people needed to add like more just flair objects like the portal and all. Just so initlevel isnt like a massive method